metaci/plan/views.py

```diff
@@ -65,10 +65,21 @@ def plan_run_repo(request, plan_id, repo_owner, repo_name):
             build = form.create_build()
             return HttpResponseRedirect(build.get_absolute_url())
     else:
-        form = RunPlanForm(plan, repo, request.user)
+        form = RunPlanForm(plan, repo, request.user, request.GET)
     context = {
         'form': form,
         'plan': plan,
         'repo': repo,
     }
     return render(request, 'plan/run.html', context=context)
+
+@login_required
```

Comment:
This should be restricted to is_staff as anyone can be a logged in user

Reply:
changed this and the other views for plan_run to match